### ISSUE TITLE
add online demo, aka playground

### ISSUE THREAD

**Issue by milahu** [justinmk/tree-sitter-ini#6]
this would be nice for quick testing of supported syntax

tree-sitter-ini is not featured in the official [tree-sitter playground](https://tree-sitter.github.io/tree-sitter/playground)

**Comment by justinmk** [comment]
hard to find any references on how to do this, but at least [this file](https://github.com/tree-sitter/tree-sitter/blob/3182efeccc5de2f50d4611466607222a29a4b059/docs/src/7-playground.md#L18) would need to be edited. It looks like only parsers owned by the tree-sitter org are added that, not third-party parsers, so I don't think this is feasible.

maybe @ObserverOfTime can confirm

**Comment by milahu** [comment]
third-party parsers can have their own playgrounds

> expected: [tree-sitter-toml playground](https://ikatyang.github.io/tree-sitter-toml/) ([source](https://github.com/ikatyang/tree-sitter-toml/tree/master/docs))

**Comment by ObserverOfTime** [comment]
[Soon™](https://tree-sitter.github.io/tree-sitter/cli/playground.html#-e--export-export_path)

**Comment by justinmk** [comment]
@milahu Does that require me to enable github pages here? That link tells me very little.

**Comment by milahu** [comment]
> github pages

yep. just copy-paste the [docs](https://github.com/ikatyang/tree-sitter-toml/tree/master/docs) folder and replace the parser

**Comment by justinmk** [comment]
That's a lot of files to manually copy/maintain, so I will wait for https://github.com/justinmk/tree-sitter-ini/issues/6#issuecomment-3628496390
